lib/promscrape/client.go

```diff
@@ -160,6 +162,18 @@ func newClient(ctx context.Context, sw *ScrapeWork) *client {
 		// See https://github.com/VictoriaMetrics/VictoriaMetrics/issues/1017#issuecomment-767235047
 		Timeout: 30 * sw.ScrapeTimeout,
 	}
+	if sw.EnableHTTP2 {
+		// golang HTTP/2 support had many problematic corner cases where
+		// dead connections would be kept and used in connection pools.
+		// https://github.com/golang/go/issues/32388
+		// https://github.com/golang/go/issues/39337
+		// https://github.com/golang/go/issues/39750
```

Comment:
It is unclear what's the purpose of this comment. What do you think it would bring for a future reader?

Reply:
Sorry for those confusing comments, they are all fixed in the past years. Remove now.